app/src/models/init-models.ts

```diff
@@ -419,8 +419,8 @@ export function initModels(sequelize: Sequelize) {
     as: "dataSourceActivityData",
     foreignKey: "activitydataId",
   });
-  CityUser.belongsTo(City, { as: "city", foreignKey: "cityId" });
-  City.hasMany(CityUser, { as: "cityUsers", foreignKey: "cityId" });
+  User.belongsToMany(City, { through: CityUser, as: "cities", foreignKey: "cityId", otherKey: "userId" });
+  City.belongsToMany(User, { through: CityUser, as: "users", foreignKey: "userId", otherKey: "cityId" });
```

Comment:
You might have to swap userId and cityId here to fix your error when calling addUser.